src/test/java/guitests/guihandles/EditPersonDialogHandle.java

```diff
@@ -0,0 +1,70 @@
+package guitests.guihandles;
+
+import guitests.GuiTestBase;
+
+public class EditPersonDialogHandle extends GuiHandle {
+
+    public static final String TITLE = "Edit Person";
+    private String githubUserNameFieldId = "#githubUserNameField";
+    private String firstNameFieldId = "#firstNameField";
+    private String lastNameFieldId = "#lastNameField";
+    private String cityFieldId = "#cityField";
+    private String tagSearchFieldId = "#tagList";
+    private String cancelButtonText = "Cancel";
+
+    public EditPersonDialogHandle(GuiTestBase guiTestBase) {
```

Comment:
I think instead of passing GuiTestBase, this should be given a reference to the FxRobot instance.